pkgs/development/python-modules/pip-chill/default.nix

```
@@ -0,0 +1,47 @@
+{ lib, fetchFromGitHub, buildPythonPackage, bandit, coverage, flake8, nose
+, pytest, sphinx,
+#ssort,
+tox, twine, watchdog, python3, bump2version, pip, setuptools, pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pip-chill";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rbanffy";
+    repo = "pip-chill";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-oWq3UWBL5nsCBUkaElashZKvm7pN3StJNubgU++8YFs=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pip
+    bandit
+    coverage
+    flake8
+    nose
+    pytest
+    sphinx
+    #ssort
+    tox
+    twine
+    watchdog
+  ] ++ [ bump2version ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
```

the tests try to run `pip-chill` as an executable, which should be patched to use the correct path:

```suggestion
  preCheck = ''
    substituteInPlace tests/test_pip_chill.py \
      --replace-fail "pip_chill/cli.py" "${placeholder "out"}/bin/pip-chill"
  '';
```